bin/src/lib.rs

```diff
@@ -840,3 +869,87 @@ mod tests {
         }
     }
 }
+
+#[cfg(not(feature = "metrics"))]
+pub struct ConfigMetrics;
```

Comment:
Does this API need to be public?

Reply:
This is no longer required to be public.